R/assess_last_30_bugs_status.R

```diff
@@ -0,0 +1,48 @@
+#' Assess how many recent BugReports have been closed
+#'
+#' @eval assess_family_roxygen(
+#'   "last_30_bugs_status",
+#'   "a logical vector indicating whether a recent BugReport was closed")
+#'
+#' @export
+assess_last_30_bugs_status <- function(x, ...) {
+  bug_reports_status(x)
+}
+
+attributes(assess_last_30_bugs_status)$column_name <- "bugs_status"
+attributes(assess_last_30_bugs_status)$label <- "vector indicating whether BugReports status is closed"
+
+
+
+bug_reports_status <- function(x, ...) {
+  UseMethod("bug_reports_status", x$bug_reports)
+}
+
+
+
+bug_reports_status.github_bug_report <- function(x, ...) {
+  pkg_metric(
+    vapply(x$bug_reports, "[[", character(1L), "state") == "closed",
+    class = "pkg_metric_last_30_bugs_status")
+}
+
+
+
+bug_reports_status.gitlab_bug_report <- function(x, ...) {
+  pkg_metric(
+    vapply(x$bug_reports, "[[", character(1L), "state") == "closed",
+    class = "pkg_metric_last_30_bugs_status")
+}
+
+
+
+#' Score a package for number of recently opened BugReports that are now closed
+#'
+#' @eval score_family_roxygen("last_30_bugs_status")
+#' @return a fractional value indicating percentage of last 30 bug reports that
```

Comment:
30 was used because it's the default size returned by the GitHub issues API. We can discuss further, but lets get this code pulled in and we can consider changes from there.